Server/Quality_of_Life_changer.Contracts/Queries/GetTodayEventsQuery.cs

```diff
@@ -1,8 +1,8 @@
 ﻿namespace Quality_of_Life_changer.Contracts.Queries;
 
 using MediatR;
-using Model.Entities;
+using Model.Calendar;
 
 public record GetTodayEventsQuery : IRequest<GetTodayEventsResponse>;
 
-public record GetTodayEventsResponse(IReadOnlyCollection<CalendarEvent> Events);
\ No newline at end of file
+public record GetTodayEventsResponse(IReadOnlyCollection<CalendarEvent> TodayEvents);
```

Comment:
use `camelCase`, not `PascalCase` for function parameters

Reply:
the official documentation recommends using `PascalCase` for `record` `class` and `struct`. [proof](https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions)